test-project/Assets/Generated/Source/improbable/gdk/tests/nonblittabletypes/SecondEventPayload.cs

```diff
@@ -4,9 +4,10 @@
 
 using System.Linq;
 using Improbable.Gdk.Core;
+using UnityEngine;
```

Comment:
do we need that?

Reply:
🤔 I don't know why I would've added it otherwise - will double check